homeassistant/components/cookidoo/coordinator.py

```diff
@@ -55,10 +56,12 @@ def __init__(
             config_entry=entry,
         )
         self.cookidoo = cookidoo
+        self.user: CookidooUserInfo | None = None
```

Comment:
Since we expect async setup to always succeed, set the type outside of the constructor and dont make it nullable